src/items/armor-item/armorSheet.ts

```diff
@@ -49,7 +52,7 @@ export class ArmorSheet extends RqgItemSheet<ItemSheet.Options, ArmorSheetData |
       data: itemData,
       armorData: itemData.data,
       sheetSpecific: {
-        allHitLocations: getHitLocations(),
+        allHitLocations: hitLocationTranslationKeys.map((key) => localize(key)),
```

Comment:
This is problematic since `getHitLocations()` reads the available hit locations from a system setting so that you can add more. There is a list of default (human) hitlocations here: https://github.com/sun-dragon-cult/fvtt-system-rqg/blob/master/src/system/settings/hitLocationNames.ts
And the starter set adds more hitlocations to that setting when you install it, see https://github.com/sun-dragon-cult/cha-rqg-fvtt-starterset/blob/master/src/scripts/moreHitlocations.mjs. Not sure how to make this translatable in a good way - I think this needs some redesign to make that possible.

Reply:
Yeah, that was a fear I had too... I think we need to load it based on supported body types (that can get added in different modules)

Let me think about it this weekend. I may be around a few nights